jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/variables/dash/DataSetProcessInstanceWithVariablesListPresenter.java

```diff
@@ -140,54 +140,31 @@ public void getData( final Range visibleRange ) {
         try {
             if ( !isAddingDefaultFilters() ) {
                 final FilterSettings currentTableSettings = dataSetQueryHelper.getCurrentTableSettings();
```

Comment:
@nmirasch could you help reviewing this bit, you're probably more familiar with this code.